src/MovieDetail/MovieDetail.css

```diff
@@ -0,0 +1,17 @@
+.movieDetail {
```

Comment:
This still needs some fine tuning and what not, but a great proof of concept for the time being.